example/app/data/ExtraDataSource/remote-package/entities/remoteEntity.json

```diff
@@ -0,0 +1,6 @@
+{
+  "_id": "remoteExample",
+  "name": "RemoteEntity",
+  "a": "test",
+  "type": "../blueprints/RemoteBlueprint"
```

Comment:
this will probably give an validation error, since the attribute "b" is missing and is required in the blueprint "RemoteBlueprint"